src/app/Api.Views/Models/Domain/CreateUserModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Api.Views.Models.Domain;
+
+public class CreateUserModel
```

Comment:
Do not see the difference between this domain model and the models in `View` folder. What do I missing?